### ISSUE TITLE
The “new follower” notification looks off

### ISSUE THREAD

**Issue by meduzen** [elk-zone/elk#1409]
This is a follow-up on [this discussion](https://elk.zone/m.nintendojo.fr/@meduz/109739613897034118) (that already led to #1408). I’m strictly focusing on the “new follower” notification, not on the grouping (I have no idea how the followers grouping looks like).

When a new follower appears in the notification screen, the visual hierarchy looks kinda broken in the sense that I can’t directly separate this notification from another one.

Looking at current Elk (0.6.1), we can see this:

<details>
<summary>screenshots of small and not small viewports</summary>
<img width="701" alt="image" src="https://user-images.githubusercontent.com/9340937/214156119-0ac9a6d5-aaf9-4c9d-9616-a96cff8c1db9.png">
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/9340937/214156031-d820d413-0432-437e-8d07-973a52992132.png">
</details>

Which gives this visual hierarchy (according to me):

![Capture d’écran 2023-01-23 à 22 58 19](https://user-images.githubusercontent.com/9340937/214159465-afe38d4e-57af-49c0-9c4d-9845bd38fcd3.png)

My perception is that the header part (header picture and “new follower” message) and the “descriptive” part (name, description, data) are not visually connected together. They feel separated, like if they were two different informations.

I think making it look more like “1 block” could help preserve some sense of structure, while maintaining the header.

For this, an option could be to group more info around the “followed you” message. Here’s a very ugly changed I pulled out by tinkering in the browser dev tools:

<details>
<summary>beware, it’s ugly</summary>
<img width="675" alt="image" src="https://user-images.githubusercontent.com/9340937/214159030-293ca4b8-35e7-4ee6-ad72-8fd58be129c5.png">
</details>

Another option that I think seems to work rather well would be to just tune the picture so it doesn’t act like a structure breaker, like this:

<img width="927" alt="image" src="https://user-images.githubusercontent.com/9340937/214161990-88376e06-501e-46ab-b95c-ca746386ffe7.png">

<details>
<summary>same in light mode</summary>
<img width="915" alt="image" src="https://user-images.githubusercontent.com/9340937/214161967-f5e2ba95-5246-4e50-b352-1148b350442c.png">
</details>

For this result ☝️, the picture wrapper receives the following styles:

```css
position: relative;
z-index: -1;
aspect-ratio: 3.4;

border-radius: 12px;
opacity: 0.5;
```

Alternatively, leaving the radius as now (~4px) but with an aspect ratio of 4 of for could also work (but I think the increased radius helps the structure):

<details>
<summary>same with higher aspect ratio and original radius</summary>
<img width="1160" alt="image" src="https://user-images.githubusercontent.com/9340937/214163634-d50e6025-f999-49af-a8a0-339ac70cb15d.png">
</details>

**Comment by ZMYaro** [comment]
I agree the header images confuse the visual hierarchy.  I would honestly be in favor of removing it from the notification entirely, as it makes the notification much taller than others.  Failing that, I like this suggestion of dimming it and making it the background behind the text.

**Comment by xPaw** [comment]
Related: #1369
